### ISSUE TITLE
NRF52832-DK (MPU9250)

### ISSUE THREAD

**Issue by fsafa001** [kriswiner/nRF51822_IoT_Breakout#3]
Hello Mr Kris, i am working with MPU9250 with the new nrf52832 development board i am having an issue with the initialization of the MPU9250 function every time i compile and upload it to the board it doesn't work the program stop running at initMPU9250() function. I was hopping if you could help me what going wrong.

**Comment by kriswiner** [comment]
Need a little more information.

Can you read the WHO_AM_I register of the MPU9250?

**Comment by fsafa001** [comment]
yes i was able to read the WHO_AM_I and the self test was working

**Comment by kriswiner** [comment]
OK, then what is the symptom? What is the specific problem besides doesn't
work or stops?

**Comment by fsafa001** [comment]
so everytime i call the initMPU9250() function nothing happens

**Comment by fsafa001** [comment]
is it possible that maybe one of the configuration inside that function is not working?

**Comment by kriswiner** [comment]
Put in some serial diagnostics then, where does it stop?
